services/core-web/src/components/Forms/reports/ReportPermitRequirementForm.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { FC } from "react";
+import { Field } from "redux-form";
+import { Button, Col, Row, Typography } from "antd";
+import { useSelector } from "react-redux";
+import {
+  FORM,
+  IMineReport,
+  IMineReportPermitRequirement,
+  IPermitCondition,
+  MINE_REPORT_SUBMISSION_CODES,
+  REPORT_FREQUENCY_HASH,
+  REPORT_MINISTRY_RECIPIENT_HASH,
+  REPORT_REGULATORY_AUTHORITY_CODES_HASH,
+  REPORT_TYPE_CODES,
+} from "@mds/common";
+import { required } from "@mds/common/redux/utils/Validate";
+import FormWrapper from "@mds/common/components/forms/FormWrapper";
+import RenderSelect from "@mds/common/components/forms/RenderSelect";
+import RenderDate from "@mds/common/components/forms/RenderDate";
+import RenderCancelButton from "@mds/common/components/forms/RenderCancelButton";
+import RenderField from "@mds/common/components/forms/RenderField";
+import RenderRadioButtons from "@/components/common/RenderRadioButtons";
+import { requiredRadioButton } from "@common/utils/Validate";
```

Comment:
would want to update the import for requiredRadioButton and RenderRadioButtons to be from @mds/common